### DISCUSSION TITLE
403 nginx error page. Macbook M1

### DISCUSSION THREAD

**Discussion by redouan-hajjari** [markshust/docker-magento#1155] [General]
Hi Guys,

Im new on docker. and i tried to understand docker. i have exist project and put that on the src page.
Everything works fine. only when i tried to find the url. it says 403 forbidden page.
when i create a normale html file. and i tried to visit that file. it says the same.

How can i fix this? and how can i test it?

I tried many time to start en stop. and i see that the status is correct.

**Comment by redouan-hajjari** [comment]
Update: 
error: Port must be configured within host parameter (like localhost:3306 

when i run 
bin/setup ******.test

**Comment by RowBowl** [comment]
Personally, when i run into a 403 error or nginx related error page, I usually have a missing vendor directory. Check if it's there, and either run bin/composer install or bin/copyfromcontainer vendor to copy an existing vendor directory from your docker container (assuming its present). Not sure if that helps
